h/groups/views.py

```diff
@@ -15,13 +15,15 @@
 
 @view_config(route_name='group_create',
              request_method='GET',
-             renderer='h:groups/templates/create.html.jinja2',
-             permission="authenticated")
+             renderer='h:groups/templates/create.html.jinja2')
 def create_form(request):
     """Render the form for creating a new group."""
     if not request.feature('groups'):
         raise exc.HTTPNotFound()
 
+    if request.authenticated_userid is None:
+        raise exc.HTTPNotFound()
```

Comment:
"we have a catch-all for unauthorized that turns it into a 404." Ah. Why?
(This relates to https://github.com/hypothesis/h/issues/2502.)

On Wed, Sep 16, 2015 at 10:44 AM, Nick Stenning <notifications@github.com>
wrote:

> In h/groups/views.py
> <https://github.com/hypothesis/h/pull/2500#discussion_r39661503>:
>
> >  def create_form(request):
> >      """Render the form for creating a new group."""
> >      if not request.feature('groups'):
> >          raise exc.HTTPNotFound()
> >
> > +    if request.authenticated_userid is None:
> > +        raise exc.HTTPNotFound()
>
> Fair enough. Same result, though. This commit doesn't change behaviour
> from before it broke.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hypothesis/h/pull/2500/files#r39661503>.
>